openmetadata-ui/src/test/java/org/openmetadata/catalog/selenium/objectRepository/Common.java

```diff
@@ -80,7 +81,42 @@
   By topics = By.xpath("(//button[@data-testid='tab'])[2]");
   By selectTable = By.xpath("(//button[@data-testid=\"table-link\"])[2]");
   By addTag = By.xpath("//div[@data-testid='tag-conatiner']");
-  By breadCrumbTag = By.xpath("(//span[@data-testid='tags'])[1]");
+  By breadCrumbTags = By.xpath("(//span[@data-testid='tags'])[1]");
+  By ingestionTab = By.xpath("//button[@data-testid='tab'][@id='ingestions']");
+  By addIngestion = By.cssSelector("button[data-testid='add-new-ingestion-button']");
+  By name = By.cssSelector("[data-testid='name']");
+
+  public By name() {
```

Comment:
The methods to get these fields are no longer needed.

They get auto-generated by the "@Getter" annotation above.

Reply:
See #2975

Reply:
Removed the methods that were no longer needed